### ISSUE TITLE
RFM - workrave

### ISSUE THREAD

**Issue by vikusz** [chocolatey-community/chocolatey-package-requests#1073]
- [x] Issue title starts with 'RFM - '
- [x] I have followed the Package Triage Process and I do NOT want to become maintainer of the package;
- [x] There is no existing open maintainer request for this package;
 
Package URL: https://chocolatey.org/packages/workrave
Package source URL: https://github.com/Claud/chocolatey-packages

Date the maintainer was contacted (in YYYY-MM-DD): 2021-08-30
How the maintainer was contacted: Through the contact maintainer link on the package website. No reply received as of 2021-09-27

**Comment by teknowledgist** [comment]
I still need to be given maintainer status in order to submit the package.  😄

**Comment by TheCakeIsNaOH** [comment]
Done.

**Comment by AdmiringWorm** [comment]
@teknowledgist what is the status of this request?

**Comment by teknowledgist** [comment]
@AdmiringWorm I am the acting maintainer at this point.  There hasn't been an approved package [because of an installer problem on older systems - i.e. the validation system](https://github.com/rcaelers/workrave/issues/367), but that should be resolved shortly with the next release.

TLDR: you can close this request.
